brightening-test/brightening-test.cpp

```diff
@@ -1,44 +1,54 @@
 #include "CppUnitTest.h"
 #include "../brightener.h"
+#include <vector>
 
 using namespace Microsoft::VisualStudio::CppUnitTestFramework;
+using namespace ImageProcessor;
 
 namespace brighteningtest
 {
-	TEST_CLASS(BrighteningTest)
-	{
-	public:
-		
-		TEST_METHOD(BrightensWholeImage)
-		{
-			auto image = std::make_shared<Image>(2, 2);
-			image->pixels[0] = 45; image->pixels[1] = 55;
-			image->pixels[2] = 65; image->pixels[3] = 254;
-
-			ImageBrightener brightener(image);
-			int attenuatedCount = brightener.BrightenWholeImage();
-			Assert::AreEqual(1, attenuatedCount);
-			Assert::AreEqual(90, int(image->pixels[2]));
-		}
-
-		TEST_METHOD(BrightensWithAnotherImage)
-		{
-			auto image = std::make_shared<Image>(2, 2);
-			image->pixels[0] = 45; image->pixels[1] = 55;
-			image->pixels[2] = 65; image->pixels[3] = 75;
-            ImageBrightener brightener(image);
-            
+    TEST_CLASS(BrighteningTest)
+    {
+    public:
+
+        TEST_METHOD(BrightensWholeImage)
+        {
+            auto image = CreateImage({ 45, 55, 65, 254 });
+
+            int attenuatedCount = 0;
+            BrightenWholeImage(image, attenuatedCount);
+
+            Assert::AreEqual(1, attenuatedCount);
+            Assert::AreEqual(90, int(image->m_pixels[2]));
+        }
+
+        TEST_METHOD(BrightensWithAnotherImage)
+        {
+            auto image = CreateImage({ 45, 55, 65, 75 });
+
             // Test by brightening only the right part
             auto brighteningImage = std::make_shared<Image>(2, 2);
-            brighteningImage->pixels[0] = 0; brighteningImage->pixels[1] = 25;
-            brighteningImage->pixels[2] = 0; brighteningImage->pixels[3] = 25;
+            brighteningImage->m_pixels[0] = 0; brighteningImage->m_pixels[1] = 25;
```

Comment:
Why not use `CreateImage` to create the `brighteningImage` as well?

Reply:
Done